pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -578,7 +591,10 @@ func GatherHostSubnet(i *Gatherer) func() ([]record.Record, []error) {
 		}
 		records := make([]record.Record, 0, len(hostSubnetList.Items))
 		for _, h := range hostSubnetList.Items {
-			records = append(records, record.Record{Name: fmt.Sprintf("config/hostsubnet/%s", h.Host), Item: HostSubnetAnonymizer{&h}})
+			records = append(records, record.Record{
```

Comment:
hostsubnets doesn't have namespace, because they are in Cluster scope.
You can see the scope using:
```
oc get crd hostsubnets.network.openshift.io -o=yaml | yq .spec.scope
```